ci/checks/copyright.py

```diff
@@ -18,7 +18,18 @@
 import argparse
 import io
 import os
-import gitutils
+import sys
+
+SCRIPT_DIR = os.path.dirname(
+    os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
```

Comment:
is there any reason this couldn't be replaced with something less verbose? like this:

```python
SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
```

Reply:
Ya, I just copied that from some other code I had. Is this ok?
```python
SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.expanduser(__file__)))
```
Would like to keep the `expanduser` in there since `realpath` doesn't remove that portion:
```python
>>> os.path.realpath("~/Repos/rapids/cuml-dev/cpp/build/test/../test/CMakeFiles/ml.dir/link.txt")
'/home/<USER>/Repos/rapids/cuml-dev/~/Repos/rapids/cuml-dev/cpp/build/test/CMakeFiles/ml.dir/link.txt'
```
With it the path comes out better:
```python
>>> os.path.realpath(os.path.expanduser("~/Repos/rapids/cuml-dev/cpp/build/test/../test/CMakeFiles/ml.dir/link.txt"))
'/home/<USER>/Repos/rapids/cuml-dev/cpp/build/test/CMakeFiles/ml.dir/link.txt'
```

Reply:
I would recommend checking out the StackOverflow link below. It says 

> Module `__file__` attributes (and related values) should now always contain absolute paths by default, with the sole exception of `__main__.__file__` when a script has been executed directly using a relative path. (Contributed by Brett Cannon in [bpo-18416](https://bugs.python.org/issue18416).)

So `__file__` will always either return an absolute path or relative path based on how the script is invoked. However, neither of those return values should include a `~/` which means the `os.path.expanduser` call shouldn't be necessary.

https://stackoverflow.com/a/22866630/5433976